src/main/java/picard/vcf/MergeVcfs.java

```diff
@@ -54,14 +54,74 @@
+ * Combines multiple variant files into a single variant file.
+ *
+ * <h3>Inputs</h3>
+ *  <ul>
+ *      <li>one or more input VCF files in text uncompressed (name ending with ".vcf"), compressed text compressed format (name ending with ".vcf.gz")
+ *          or binary format BCF (name ending with ".bcf"),</li>
+ *      <li>and optionally a sequence dictionary file (typically name ending in .dict) if the input VCF does not contain a
+ *          complete contig list and if the output index is to be created (true by default).</li>
+ *  </ul>
+ *  <p>
+ *  The input variant files must:
+ *     <ul>
+ *         <li>if there are samples, those must be the same across all input files,</li>
+ *         <li>input file headers must be contain compatible declarations for common annotations (INFO, FORMAT fields) and filters.</li>
+ *         <li>and input files variant records must be sorted by their contig and position following the sequence dictionary provided
+ *         or the header contig list.</li>
+ *     </ul>
+ * </p>
+ * <p>
+ *     You can either directly specify the list of files by specifying <code>INPUT</code> multiple times, or provide a list
+ *     in a file with name ending in ".list" to <code>INPUT</code>.
+ * </p>
+ *
+ * <h3>Outputs</h3>
+ * A VCF or BCF sorted by contig and then by position in the contig following the dictionary provided in the input.
+ * <p>
+ *     Also an index file will be produced along-side the output variant file unless
+ * the user un-sets the <code>CREATE_INDEX</code> argument.
+ * </p>
+ * <p>
+ *     The format used for the output is determined by the output file name's ending (.vcf, .vcf.gz or .bcf).
+ * </p>
+ *
+ * <h3>Usage examples</h3>
+ * <h4>Examples 1:</h4>
+ *     We combine several variant files in different formats, where at least one of them contains the contig list in its header.
+ *     The output file format is gzip compressed VCF (file name ending in ".vcf.gz"). Alongside the out
+ *     vcf an index file will be created with the name <code>output.vcf.gz.tbi</code>:
+ * <pre>
+ *     java -jar picard.jar MergeVcfs \
+ *          I=input_variants.01.vcf \
+ *          I=input_variants.02.vcf \
+ *          I=input_variants.03.vcf.gz \
+ *          I=input_variants.04.bcf \
+ *          O=output_variants.vcf.gz
+ * </pre>
+ * <h4>Example 2:</h4>
+ *     Similar to example 1 above but this time we don't want the index to be created and
+ *     we explicitly specify a sequence dictionary:
+ * <pre>
+ *     java -jar picard.jar MergeVcfs \
+ *          D=reference.dict \
+ *          I=input_variants.01.vcf \
+ *          I=input_variants.02.vcf \
+ *          I=input_variants.03.vcf.gz \
+ *          I=input_variants.04.bcf \
+ *          CREATE_INDEX=false \
+ *          O=output_variants.vcf.gz
+ * </pre>
+ * <h4>Example 3:</h4>
+ *      Similar to example 1 but we use a input list file to specify the input files:
+ * <pre>
+ *     java -jar picard.jar MergeVcfs \
```

Comment:
Perhaps move this simplest command to the top. Example commands could be in decreasing order of popularity.

Reply:
It's not actually the simplest conceptually since it uses a list instead of passing in file names directly. I like it being last.